docker/Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM docker.montagu.dide.ic.ac.uk:5000/orderly.server:51680f5
 
-RUN apt-get update || apt-get update && apt-get install -y git
+RUN apt-get update || apt-get update && apt-get -y install \ 
```

Comment:
with 1907 merged you can drop the first `apt-get update ||` part of this.  `git` is also unnecessary as it's in the base image